packages/react-components/react-table/src/hooks/useKeyboardResizing.ts

```diff
@@ -11,16 +11,16 @@ const PRECISION_FACTOR = 1 / 4;
 export function useKeyboardResizing(columnResizeState: ColumnResizeState) {
   const [columnId, setColumnId] = React.useState<TableColumnId>();
   const onChangeRef = React.useRef<EnableKeyboardModeOnChangeCallback>();
-  const addListenerTimeout = React.useRef<number>();
+  const { findPrevFocusable } = useFocusFinders();
 
   const columnResizeStateRef = React.useRef<ColumnResizeState>(columnResizeState);
   React.useEffect(() => {
     columnResizeStateRef.current = columnResizeState;
   }, [columnResizeState]);
 
-  const { targetDocument } = useFluent();
+  const refMap = React.useMemo(() => new Map(), []);
```

Comment:
Let's be more specific with naming here `resizeHandleRefs` or smth like that

Reply:
Also the map is not typed at all - everything is working because it's a Map of `any` 😱😱

how about
```suggestion
  const [refMap] = React.useState() => new Map<TableColumnId, React.RefObject<HTMLDivElement>>());
```

useState might also be a better option because there is a guarantee that it will only be created once. Take a look at the [caveats of useMemo](https://react.dev/reference/react/useMemo#caveats)